.github/workflows/release.yml

```diff
@@ -96,6 +96,20 @@ jobs:
       - name: Run build script
         run: |
           cd build/android && printf "y" | ./build.sh release
+      - name: Sign sample-gltf-viewer
+        run: |
+          echo "${APK_KEYSTORE_BASE64}" > filament.jks.base64
+          base64 --decode filament.jks.base64 > filament.jks
+          BUILD_TOOLS_VERSION=30.0.3
```

Comment:
Do we need to do anything to ensure we have that version installed? Could we just take the most recent that's already installed instead?

Reply:
According to https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md#android, it should be pre-installed on macOS runners. I don't know of any way to just use the latest version

Reply:
I would just ls the directory, sort, uniq and take the first item. Here's how I used to do it for the NDK:

```
local ndk_version=$(ls "${ndk_side_by_side}" | sort -V | tail -n 1 | cut -f 1 -d ".")
```

Reply:
Cool, I'll try that.